mods/d2k/weapons.yaml

```diff
@@ -9,74 +9,95 @@ LMG:
 		ContrailDelay: 0
 		ContrailUsePlayerColor: true
 	Warhead@1Dam: SpreadDamage
-		Spread: 96
-		Damage: 15
+		Spread: 48
+		Falloff: 100, 100, 65, 30, 0, 0, 0
```

Comment:
I see, all examples of Falloff in the yamls seemed to have the same amount of numbers so I stuck to that.